ui/Screen/NetworkDiagnostics/StateTable.svelte

```diff
@@ -24,7 +25,9 @@
     {#each Object.keys(state).sort() as urn}
       <tr>
         <td>
-          <StyledCopyable value={urn} truncate={true} expandable={false} />
+          <Copyable title="Radicle ID" clipboardContent={urn}>
```

Comment:
See my comment about the device ID.

Reply:
c4b23c1c51d798db78baeb055b138589ce938436